src/com/mostc/pftt/model/core/PhpIni.java

```diff
@@ -141,14 +143,34 @@ public PhpIni clone() {
 	}
 	
 	static final Pattern PAT_PWD = Pattern.compile("\\{PWD\\}");
+	static final Pattern PAT_TMP = Pattern.compile("\\{TMP\\}");
 	static final Pattern PAT_BS = Pattern.compile("\\\\");
 	static final Pattern PAT_FS = Pattern.compile("/");
-	public PhpIni(String ini_str, String pwd) {
+	public PhpIni(String ini_str, String pwd, String tmp) {
 		this();
 		if (pwd!=null&&ini_str.contains("{PWD}")) {
-			ini_str = StringUtil.replaceAll(PAT_PWD, pwd, ini_str);
-			
-			// CRITICAL: ensure that correct \\s are used for paths on Windows
+			if(SystemUtils.IS_OS_WINDOWS)
+			{
+				// CRITICAL: ensure that correct \\s are used for paths on Windows
+				String escapedPwd = StringUtil.replaceAll(PAT_BS, "\\\\\\\\", pwd);
```

Comment:
I think it would be more efficient and more readable to use `String.replace()` here, since we there's no need to use regexps. Also we don't need `escapedPwd`, but could modify `pwd` instead.